google/cloud/grpc_error_delegate.h

```diff
@@ -42,7 +43,7 @@ google::cloud::Status MakeStatusFromRpcError(grpc::StatusCode code,
  * libraries in `google-cloud-cpp` represent these errors using a
  * `google::cloud::Status`.
  */
-google::cloud::Status MakeStatusFromRpcError(google::rpc::Status const& status);
+google::cloud::Status MakeStatusFromRpcError(google::rpc::Status const& proto);
```

Comment:
This becomes a public API, I think I made a mistake when I made `MakeStatusFromRpcError()` public, can we avoid adding the overload to the public API too?

Reply:
This was already a public API. I just renamed the param to `proto`.

Reply:
Doh!